### PR TITLE
doc: make the 2^x requirement for msg_queue more visible

### PR DESCRIPTION
**Pull request by miri64**
My reaction to https://lists.riot-os.org/pipermail/users/2016-October/000695.html

### CODE REVIEW

File: core/include/msg.h [hunk 1]
```diff
@@ -365,6 +365,8 @@ int msg_avail(void);
 /**
  * @brief Initialize the current thread's message queue.
  *
+ * @pre @p num **must be a power of two**
```
Comment (OlegHahm) [inline core/include/msg.h]: Shouldn't we remove the second line of the parameter documentation, then?

**Comment by OlegHahm** [comment]
I'm not sure that duplicating this information will help a lot in order to reduce the questions on the mailing list/issue tracker, because many people tend just not to read the documentation anyway. However, it is worth a try.

File: core/include/msg.h (hunk 1 above)
Comment (OlegHahm) [inline core/include/msg.h]: And I would keep the upper case here.

**Comment by Lotterleben** [comment]
Well that sounds fatalistic :D

**Comment by miri64** [comment]
> I'm not sure that duplicating this information will help a lot in order to reduce the questions on the mailing list/issue tracker, because many people tend just not to read the documentation anyway. However, it is worth a try.

That's why I also added it to the boiler plate tutorials ;-)

File: core/include/msg.h (hunk 1 above)
Comment (miri64) [inline core/include/msg.h]: Done

**Comment by PeterKietzmann** [comment]
This should go in today, right? Please squash your second commit

**Comment by miri64** [comment]
Squashed.

**Pull request merged**